File: public/docs/_examples/dependency-injection/dart/web/main_1.dart
// **WARNING**
// To try out this version of the app, ensure that you update:
// - web/index.html
// - pubspec.yaml
// to refer to this file instead of main.dart

import 'package:angular2/platform/browser.dart';

import 'package:dependency_injection/app_component_1.dart';
import 'package:dependency_injection/heroes/hero_service_1.dart';

void main() {
  bootstrap(AppComponent);
}

void main_alt() {
  // #docregion bootstrap-discouraged
  bootstrap(AppComponent,
    [HeroService]); // DISCOURAGED (but works)
  // #enddocregion bootstrap-discouraged
}

File: public/docs/_examples/router-deprecated/ts/app/main.1.ts
/* First version */
// #docplaster

// #docregion all
import { bootstrap }        from '@angular/platform-browser-dynamic';
import { ROUTER_PROVIDERS } from '@angular/router-deprecated';

import { AppComponent }     from './app.component';

// #enddocregion all

/* Can't use AppComponent ... but display as if we can
// #docregion all
bootstrap(AppComponent, [
// #enddocregion all
*/

// Actually use the v.1 component
import { AppComponent as ac } from './app.component.1';
bootstrap(ac, [
// #docregion all
  ROUTER_PROVIDERS
]);
// #enddocregion all
